kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/JsonUtil.java

```diff
@@ -16,18 +16,56 @@
 
 package org.kitesdk.data.spi;
 
+import com.fasterxml.jackson.core.JsonFactory;
```

Comment:
I opened [CDK-879](https://issues.cloudera.org/browse/CDK-879) to track this. I'd rather do it in a follow-up issue because there are already components that use Jackson so it isn't a new problem and might run into issues that aren't related to this work.